### PR TITLE
Added ukrainian translation

### PR DESCRIPTION
**Pull request by kshehadeh**
# Issue

What issue is this PR targeting? If there is no issue that addresses the problem, please open a corresponding issue and link it here.

## Tasklist

 - [ ] Add tests
 - [ ] Add #fixes with the issue number that this PR addresses
 - [ ] Update the docs with any new request parameters or changes to behavior described
 - [ ] Update the [changelog](CHANGELOG.md)

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

File: locales/uk-UA.json [hunk 1]
```diff
@@ -0,0 +1,2149 @@
+{
+  "posix_locale": "uk_UA.UTF-8",
+  "aliases": "ukua",
```
Comment (dgearhart) [inline locales/uk-UA.json:3]: aliases should be an array - the value should be "uk"

Comment (kshehadeh) [reply locales/uk-UA.json]: Good catch.  Fixed.

**Review by dgearhart — COMMENTED** [review]
Also noticed that `da-DK.json` aliases needs to be an array too

**Comment by kshehadeh** [comment]
> Also noticed that `da-DK.json` aliases needs to be an array too

Can I include that fix in this PR?

**Review by dgearhart — COMMENTED** [review]
@kshehadeh should we have a simple test that verifies that the locale code and alias work - that we get back Ukrainian - just a simple gurka test?

**Comment by dgearhart** [comment]
> Can I include that fix in this PR?

@kshehadeh I think that would fine - or it could be a separate one

**Comment by kshehadeh** [comment]
> @kshehadeh should we have a simple test that verifies that the locale code and alias work - that we get back Ukrainian - just a simple gurka test?

That sounds good.  Maybe you can help me wit that? That would be my first :)

**Review by merkispavel — COMMENTED** [review]
I looked through the translation: didn't found anything illegal

**Review by dgearhart — APPROVED** [review]
:shipit:

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,7 +18,7 @@
    * FIXED: Fix way_ids -> graph_ids conversion for complex restrictions: handle cases when a way is split into multiple edges [#2848](https://github.com/valhalla/valhalla/pull/2848)
    * FIXED: Honor access mode while matching OSMRestriction with the graph [#2849](https://github.com/valhalla/valhalla/pull/2849)
    * FIXED: Fix compilation errors when boost < 1.68 and libprotobuf < 3.6  [#2878](https://github.com/valhalla/valhalla/pull/2878)
-
+
```
Comment (mandeepsandhu) [inline CHANGELOG.md:21]: can we remove the whitespace?

Comment (kshehadeh) [reply CHANGELOG.md:21]: Done

**Review by dgearhart — APPROVED** [review]
:ship:

**Pull request merged**